assets/styles/global/_labeled-input.scss

```diff
@@ -11,7 +11,7 @@
     transition-property: transform, font-size;
     transition-duration: 0.1s;
     transition-timing-function: ease-in-out;
-    z-index: 1;
+    z-index: 10;
```

Comment:
All z-indexes should use the [z-index()](https://github.com/rancher/dashboard/blob/master/assets/styles/base/_functions.scss#L29-L36) function with a key from [the list](https://github.com/rancher/dashboard/blob/master/assets/styles/base/_variables.scss#L6) so that there's a central place to see and reorder the stack.